what4/test/SymFnTests.hs

```diff
@@ -0,0 +1,227 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE LambdaCase #-}
+
+module SymFnTests where
+
+import           Control.Monad.IO.Class ( MonadIO, liftIO )
+
+import           Data.Parameterized.Classes ( ShowF(..) )
+import           Data.Parameterized.Context ( pattern (:>), (!) )
+import qualified Data.Parameterized.Context as Ctx
+import           Data.Parameterized.Nonce
+import           Data.Parameterized.Some
+import           Data.Parameterized.TraversableFC
+import qualified Data.Text as T
+import qualified Data.Map as Map
+import qualified Data.Map.Ordered as OMap
+import           Hedgehog
+import qualified LibBF as BF
+
+-- import qualified What4.Utils.Log as Log
+import           Test.Tasty
+import           Test.Tasty.Hedgehog
+import           SerializeTestUtils
+import qualified What4.Expr.Builder as S
+import           What4.BaseTypes
+import qualified What4.Interface as WI -- ( getConfiguration )
+-- import qualified What4.Serialize.Normalize as WN
```

Comment:
Can these stray comments be removed? I don't think it makes sense to keep them around, as there isn't any commented-out code that uses them.

Reply:
Removed